spark-mongodb/src/main/scala/com/stratio/provider/mongodb/reader/MongodbReader.scala

```diff
@@ -125,7 +125,7 @@ class MongodbReader(
   }
 
   private def convertToStandardType(value: Any): Any = {
-    if (value.isInstanceOf[UTF8String])
+    if (value.isInstanceOf[StringType])
```

Comment:
not sure if this should be StringType or org.apache.spark.unsafe.types.UTF8String